superduper/backends/ibis/data_backend.py

```diff
@@ -132,7 +133,10 @@ def insert(self, table_name, raw_documents):
 
     def drop_outputs(self):
         """Drop the outputs."""
-        raise NotImplementedError
+        for table in self.conn.list_tables():
+            logging.info(f"Dropping table: {table}")
+            if '_outputs__' in table:
```

Comment:
Please change this to `table.startswith(CFG.output_prefix)`

Reply:
Ah yes good point.